bindings/pyroot_experimental/PyROOT/python/ROOT/_facade.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, module):
         self.gROOT = gROOT
 
         # Expose some functionality from CPyCppyy extension module
-        cppyy_exports = [ 'Double', 'Long', 'nullptr' ]
+        cppyy_exports = [ 'Double', 'Long', 'nullptr', 'addressof', 'bind_object' ]
```

Comment:
addressof (at least) is around since a while, necessary in some cases and unfortunately not documented. Perhaps we need to do something about this.

Reply:
Yes. I will for sure document everything that is in the API, but for these guys I still do not know exactly how to expose it. Their docs won't go in the Doxygen because they do not correspond to any C++ class in ROOT. We will need to have some docs taken from the ROOT Python module itself, as things we (still) expose from Cppyy.